src/open_inwoner/pdc/tests/test_data_migrations.py

```diff
@@ -1,106 +0,0 @@
-from django.contrib.gis.geos import Point
```

Comment:
Was there a specific reason we had to let these go?

If the amount of migrations is an issue we could look into squashing the migrations. IIRC it also had some complications with data migrations.